testsuite/forge/src/backend/k8s/cluster_helper.rs

```diff
@@ -0,0 +1,310 @@
+// Copyright (c) The Diem Core Contributors
```

Comment:
these have just been moved from the swarm.rs file correct?

Reply:
yes, I move all cluster operations to here with some refactory and buildup wrappers upon in swarm